Dockerfile-periodic

```diff
@@ -28,11 +21,17 @@ COPY ./tools /wsf_scraper/tools
 COPY ./entrypoint.sh /wsf_scraper/entrypoint.sh
 
 COPY ./scrapy.cfg /wsf_scraper/scrapy.cfg
-COPY ./requirements-periodic-scraping.txt /wsf_scraper/requirements.txt
+
+# Pipenv require these two files to install requirements in the pinned version
+COPY ./Pipfile /wsf_scraper/Pipfile
```

Comment:
Do you need to copy the Pipfile into the container, or just the lock file?

(I don’t know much about pipenv, so this may be correct, but it feels wrong to me.)

Reply:
From what I read online, we need to copy the Pipfile as well